OpenRA.Mods.Common/Traits/BotModules/SupportPowerBotModule.cs

```diff
@@ -136,6 +135,9 @@ void IBotTick.BotTick(IBot bot)
 
 			var map = world.Map;
 			var checkRadius = powerDecision.CoarseScanRadius;
+			var suitableLocations = new List<int[]>();
```

Comment:
Style nit: using an int array here is a bit odd.

This would be cleaner as:
```csharp
var suitableLocations = new List<(MPos UV, int Attractiveness)>();
```

and then you can use
```csharp
suitableLocations.Add((tl, consideredAttractiveness));
```

and `location.Attractiveness`, `location.UV.ToCPos(map)`.